weblate/templates/translation-download.html

```diff
@@ -1,23 +1,23 @@
 {% load i18n %}
 {% load permissions %}
 
-{% perm 'upload.perform' object as user_can_upload_translation %}
+{# perm 'upload.perform' object as user_can_upload_translation %}
```

Comment:
Is this intentional?

Reply:
For now, yes. There was a problem with this LOC that seemed unreasonably difficult to resolve. Surely will be commented in again.